autoswitch_virtualenv.plugin.zsh

```diff
@@ -179,7 +179,7 @@ function _default_venv()
         _maybeworkon "$(_virtual_env_dir "$AUTOSWITCH_DEFAULTENV")" "$venv_type"
     elif [[ -n "$VIRTUAL_ENV" ]]; then
         local venv_name="$(_get_venv_name "$VIRTUAL_ENV" "$venv_type")"
-        _autoswitch_message "Deactivating: ${BOLD}${PURPLE}%s${NORMAL}\n" "$venv_name"
+        _autoswitch_message "Deactivating: ${BOLD}${PURPLE}$venv_name${NORMAL}"
```

Comment:
you removed a `\n` here which is causing the messages to display correctly. Could you re-add it?

Reply:
I will.